amundsen_application/static/js/pages/DashboardPage/index.tsx

```diff
@@ -205,6 +205,16 @@ export class DashboardPage extends React.Component<
           </div>
           {/* <div className="header-section header-links">links here</div> */}
           <div className="header-section header-buttons">
+            <a
+              id="dashboard-link"
```

Comment:
Could you use data-type instead of using the id?

Reply:
hey @Golodhros , pr is updated, could you share a bit more on when to use which one as I mostly follow the initial button?

Reply:
Thanks a lot Tao!

We should always use `data-` attributes to add information to the DOM. The previous implementation using an id wasn't the right way of doing things. From now on, we should always try to use data attributes. In modern frontend development, there is almost no reason to use an id attribute.

Reply:
got it, thanks for the info!